src/js/store.js

```diff
@@ -98,4 +98,6 @@ export const timestamp = writable(0);
 export const faviconURL = writable('/48x48.png');
 export const availableMchadUsers = writable([]);
 export const spotlightedTranslator = writable(null);
+export const displayMode = writable(paramsEmbedded ? DisplayMode.EMBEDDED : DisplayMode.FULLPAGE);
```

Comment:
This seems like it should be a constant value though. You only set it once before the app is mounted. Maybe try moving this to constants and getting the page type by setting a ``window`` value before the app mounts or by checking the url.

Reply:
Nvm, this is better.